app/src/features/projects/components/__snapshots__/ProjectFundingItemForm.test.tsx.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ProjectFundingItemForm renders correctly with default empty values 1`] = `
+<div
+  id="root"
+/>
+`;
+
+exports[`ProjectFundingItemForm renders correctly with existing funding item values with agency_id 1 1`] = `
+<div
+  id="root"
+/>
+`;
```

Comment:
these look incorrect to me... they all have same snapshot, is that expected??

Reply:
No this is not right, just noticed it after hitting the re-review button, working on it now